### PR TITLE
[FLINK-12775] bump up flink-shaded dependencies version to 7.0

### PR DESCRIPTION
**Pull request by aloyszhang**
## What is the purpose of the change

This PR is to bump up flink-shaded dependencies version to 7.0 , since the flink-shaded project has release 7.0.

## Brief change log

  - bump up flink-shaded dependencies version to 7.0

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (**yes** / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (**yes** / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)

### CODE REVIEW

**Comment by aloyszhang** [comment]
@zentol

**Comment by sunjincheng121** [comment]
And I also agree to merge this PR first. What do you think?

**Comment by aloyszhang** [comment]
@sunjincheng121  Thanks for reply.  I didn't see you PR about flink-shaded-hadoop upgrade since I just pay attention to  flink-shaded-jackson which I upgraded in flink-shade project.

**Comment by aloyszhang** [comment]
@sunjincheng121  Merge this PR first looks good to me.

File: flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/AbstractHandler.java [hunk 1]
```diff
@@ -123,7 +124,7 @@ protected void respondAsLeader(ChannelHandlerContext ctx, RoutedRequest routedRe
 			} else {
 				try {
 					ByteBufInputStream in = new ByteBufInputStream(msgContent);
-					request = MAPPER.readValue(in, untypedResponseMessageHeaders.getRequestClass());
+					request = MAPPER.readValue((InputStream) in, untypedResponseMessageHeaders.getRequestClass());
```
Comment (zentol) [inline flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/AbstractHandler.java]: you can solve this in a nicer way by changing the line above to `InputStream in = new ByteBufInputStream(msgContent);`

**Pull request merged**